ci-operator/jobs/openshift/cluster-autoscaler-operator/openshift-cluster-autoscaler-operator-periodics.yaml

```diff
@@ -126,7 +126,7 @@ periodics:
       - name: JOB_NAME_SAFE
         value: e2e-gcp-operator
       - name: TEST_COMMAND
-        value: make test-e2e
+        value: cd /go/src/github.com/openshift/cluster-autoscaler-operator && make test-e2e
```

Comment:
Can we change them all to make -C <DIR> test-e2e then there's no issue with popping directories.

Reply:
done